main.py

```diff
@@ -0,0 +1,42 @@
+"Python Consumer"
+from confluent_kafka import Consumer
+
+KAFKA_TOPIC = "driver-positions"
+
+print("Starting Python Consumer.")
+
+# Configure the group id, location of the bootstrap server,
+# Confluent interceptors
+consumer = Consumer({
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code contains a resource that might not be closed properly. This can cause a resource leak that slows down or crashes your system.

**Fix**
Consider closing the following resource: *consumer*. The resource is allocated by call *confluent_kafka.Consumer*. The resource is referenced at lines 18, 23, 42. The resource is closed at line 42. There are other execution paths that do not contain closure statements, for example, when *subscribe()* throws an exception. Close *consumer* in a try-finally block to prevent the resource leak.